signalstickers/core/admin/pack_admin.py

```diff
@@ -98,40 +110,90 @@ def get_form(self, request, obj=None, **kwargs):  # pylint: disable=arguments-di
         )
         return super().get_form(request, obj, **kwargs)
 
-    actions = ["approve"]
-
-    def approve(self, request, queryset):
-        if isinstance(queryset, Model):
-            obj = queryset
-            obj.status = PackStatus.ONLINE.name
-            obj.save()
-            updated_count = 1
-        else:
-            updated_count = queryset.update(status=PackStatus.ONLINE.name)
-
-        msg = f"Marked {updated_count} packs as online"
-        self.message_user(request, msg, messages.SUCCESS)
-
-    approve.short_description = "Change selected packs status to online"
+    # customs actions
+    @admin.action(description="Start a bulk session")
+    def bulk_review_packs(self, request, queryset):
+        # Add the next pack ids to the GET parameters
+        ordered_queryset = queryset.order_by("id")
+        object_id = str(ordered_queryset.first().id)
+        extra_context = [
+            str(pack_id) for pack_id in ordered_queryset.values_list("id", flat=True)
+        ]
+        extra_context.remove(object_id)  # remove current object from the list
+
+        # create the GET parameters
+        params_dict = {
+            "_changelist_filters": urlencode(request.GET),
+            "next": ",".join(extra_context),
```

Comment:
Change `extra_context` var name, and use something like `_bulkreview_next` as GET param